PHPCI/Command/CreateBuildCommand.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * PHPCI - Continuous Integration for PHP
+ *
+ * @copyright    Copyright 2014, Block 8 Limited.
+ * @license      https://github.com/Block8/PHPCI/blob/master/LICENSE.md
+ * @link         https://www.phptesting.org/
+ */
+
+namespace PHPCI\Command;
+
+use PHPCI\Helper\Lang;
+use PHPCI\Service\BuildService;
+use PHPCI\Store\ProjectStore;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Create build command - creates a build for a project
+ * @author       Jérémy DECOOL (@jdecool)
+ * @package      PHPCI
+ * @subpackage   Console
+ */
+class CreateBuildCommand extends Command
+{
+    /**
+     * @var ProjectStore
+     */
+    protected $projectStore;
+
+    /**
+     * @var BuildService
+     */
+    protected $buildService;
+
+    /**
+     * @param ProjectStore $projectStore
+     */
+    public function __construct(ProjectStore $projectStore, BuildService $buildService)
+    {
+        parent::__construct();
+
+        $this->projectStore = $projectStore;
+        $this->buildService = $buildService;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('phpci:create-build')
+            ->setDescription(Lang::get('create_build_project'))
+            ->addArgument('projectId', InputArgument::REQUIRED, Lang::get('project_id_argument'))
+            ->addOption('commit', null, InputOption::VALUE_OPTIONAL, Lang::get('commit_id_option'))
+            ->addOption('branch', null, InputOption::VALUE_OPTIONAL, Lang::get('branch_name_option'));
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        $projectId = $input->getArgument('projectId');
+        $commitId = $input->getOption('commit');
+        $branch = $input->getOption('branch');
+
+        $project = $this->projectStore->getById($projectId);
+        if (empty($project)) {
+            throw new \Exception('Project does not exist:' . $projectId);
```

Comment:
I prefer to have [```InvalidArgumentException```](http://php.net/manual/it/class.invalidargumentexception.php) here.
Add space after comma to separate ```projectId``` value from message string.